concepts/05 UI Components/zz Common/15 Advanced/40 3rd-Party Frameworks Integration API/10 Create and Configure a UI Component.md

```diff
@@ -8,6 +8,7 @@ To create a UI component, call its constructor passing the target element and a
     let instance = new Accordion(element, options);
     ...
 
```

Comment:
I have the following comments regarding Line 1 above:

1) `To create a UI component, call its constructor passing the target element and a configuration object.`

For my understanding, does the developer pass the target element and a configuration object as _parameters_?
If so, do we need to specify this in the sentence?

2) `The target element may need to have additional style properties set, for example, data visualization UI components' **display** should be set to "block".`

I'd like to update this sentence so that it specifies _who_ sets the style properties. Is it the developer ("you")? For example:

`You may need to set additional properties for the target element (for example, data visualization UI components' **display** property should be set to "block").`

Reply:
1. Not really. In this case, constructor is called implicitly so we do not pass the parameters.
2. Your suggestion fits perfectly - 
You may need to set additional properties for the target element (for example, the **display** property of data visualization UI components should be set to "block").

Reply:
Thank you. Can you please apply my suggestion?

Reply:
sure

Reply:
Thank you!